modules/ROOT/pages/configure-tls-context-rtf-lb.adoc

```diff
@@ -5,9 +5,52 @@ endif::[]
 
 All inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS. When enabling inbound traffic, you must provide a valid TLS certificate. When you configure inbound traffic for Runtime Fabric, you can configure the TLS server by importing a Privacy-Enhanced Mail (PEM) or Java Keystore (JKS) file, or by importing a TLS context from the secrets manager (advanced use case).
 
-Before you can import the TLS context from the secrets manager, you must add the TLS context. 
+== Before You Begin
 
-== Add the TLS Context to a Secrets Group
+You must have the following permissions to enable inbound traffic to Runtime Fabric:
+
+* Grant access to secrets
+* Manage Secret Groups
+* Read secrets metadata
+* Write secrets
+
+These permissions are available from the Secrets Manager tab in Anypoint Access Management.
+
+. From Anypoint Platform, select *Access Management*.
++
+If you do not have access to *Access Management*, ask your organization admin (system administrator) to provide the "Manage Runtime Fabrics" permission to your user. If you have access, execute the following steps to add the permission:
+. Select *Users*.
+. Locate the row containing your username and then select the blue link in the *Username* column.
+. In the *Runtime Manager* tab, enable the *Manage Runtime Fabrics* permission for your Runtime Fabric environment.
++
+[NOTE]
+It can take up to five minutes for any permissions you add to propagate.
+
+== TLS Context Configuration Prerequisites
+
+You must perform the following configuration steps prior to adding a TLS context to a secrets group:
+
+*** Have a TLS certificate with the desired Common Name set to route traffic to Runtime Fabric.
+
+.... Create a Certificate Signing Request (CSR) for a Certificate Authority (CA). To streamline this step, larger organizations have a portal/internal service that provides certificate signing from a CA or might repurpose existing certificates. If neither of the previous options are available, follow the steps required by your CA to obtain your certificate, which usually includes the following tasks:
```

Comment:
I'm not sure how much guidance we should be providing for obtaining a certificate. Don't we still expect most customers to already have the certificates they'll use?